tests/substrate_tests/builtins.rs

```diff
@@ -755,3 +755,23 @@ fn hash() {
 
     runtime.function("test_encoding", vec![]);
 }
+
+#[test]
+fn call_chain_extension() {
+    let mut runtime = build_solidity(
+        r##"
+        import "substrate";
```

Comment:
Suggestion:

You could do `import {chain_extension as ChainExtension} from "substrate";` and test if that that works.